mutiny-core/src/lib.rs

```diff
@@ -2098,6 +2098,8 @@ impl<S: MutinyStorage> MutinyWallet<S> {
             self.ensure_mutiny_nwc_profile(subscription_client, autopay)
                 .await?;
 
+            // FIXME: switch the subscription from disabled to enabled if it was disabled
```

Comment:
need to do this

Reply:
We don't _need_ to right now.

Reply:
https://github.com/MutinyWallet/mutiny-node/issues/1174